lua/cfc_ulx_commands/curse/cl_utils.lua

```diff
@@ -1,4 +1,85 @@
------ SETUP ----
+CFCUlxCurse = CFCUlxCurse or {}
+CFCUlxCurse._marchFolderCache = CFCUlxCurse._marchFolderCache or {}
+CFCUlxCurse._marchFolderCacheDroppedRoot = CFCUlxCurse._marchFolderCacheDroppedRoot or {}
+
+
+local marchFolderCache = CFCUlxCurse._marchFolderCache
+local marchFolderCacheDroppedRoot = CFCUlxCurse._marchFolderCacheDroppedRoot
+
+local fileFind = file.Find
+local tableInsert = table.insert
+local stringSub = string.sub
```

Comment:
Our preferred format for localized functions actually goes against the style guidelines
```suggestion
local file_Find = file.Find
local table_insert = table.insert
local string_sub = string.sub
```